frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx

```diff
@@ -72,16 +77,22 @@ export const MultilineUtilizationItem: React.FC<MultilineUtilizationItemProps> =
         domain={{ x: [startDate, endDate] }}
         loading={!error && isLoading}
         query={queries.map((q) => q.query)}
-        xAxis={false}
+        xAxis={!!isOCSRendered}
```

Comment:
```suggestion
        xAxis={xAxis}
```
pass `xAxis` as a prop instead.